### DISCUSSION TITLE
Any plan to release the next-generation whisper?

### DISCUSSION THREAD

**Discussion by zh-plus** [openai/whisper#1624] [General]
I'm not rushing, I'm just wondering when the next version of the whisper model will be available. We are eager to address the issue of hallucination, which may require updated models and algorithms.

**Comment by Arputikos** [comment]
Yes, that would be great, and maybe separate models for languages, or at least any info on how to prune a model and leave just 1-2 languages

**Comment by RaulKite** [comment]
Maybe @jongwook knows anything about this :-)
